src/common/mfu_daos.c

```diff
@@ -1571,35 +1549,32 @@ static int serialize_recx_array(struct hdf5_args *hdf5,
             }
             /* created attribute in HDF5 file with encoded
              * dataspace for this record extent */
-            memset(attr_name, 64, sizeof(attr_name));
-            memset(number_str, 16, sizeof(number_str));
-            memset(attr_num_str, 16, sizeof(attr_num_str));
-            path_len = snprintf(number_str, 10, "%d",
-                        (int)(*ak_index));
-            if (path_len >= 16) {
+            memset(attr_name, ATTR_NAME_LEN, sizeof(attr_name));
+            memset(number_str, ATTR_NAME_LEN, sizeof(number_str));
+            memset(attr_num_str, ATTR_NAME_LEN, sizeof(attr_num_str));
```

Comment:
@dsikich Is this what you meant?
```suggestion
            memset(attr_name, 0, ATTR_NAME_LEN);
            memset(number_str, 0, ATTR_NAME_LEN);
            memset(attr_num_str, 0, ATTR_NAME_LEN);
```

Reply:
@daltonbohning oof, thanks for catching that. Yes, that is what I meant to do.